app/templates/components/travis-form.hbs

```diff
@@ -0,0 +1,10 @@
+<form onsubmit={{action 'submit' }} class="travis-form">
+  {{yield (hash
+    validators=validators
+    hasErrors=hasErrors
+    isValid=isValid
+    submit=(action 'submit')
+    input=(component 'forms/form-field' form=this fieldComponentName='forms/form-input')
+    textarea=(component 'forms/form-field' form=this fieldComponentName='forms/form-textarea')
+  )}}
+</form>
```

Comment:
@backspace thanks, I'm happy that you like it :) I was thinking about refactoring other forms too, but thought that it would inflate this PR. So I created a separate issue: https://github.com/travis-pro/team-teal/issues/2898 - I'm planning to implement it right after the new help page project.

Reply:
Great! I agree that it’s better to keep separated.